bot/admin/admin.py

```diff
@@ -109,6 +110,29 @@ async def cmd_for_bot_stuff(self, ctx: commands.Context):
         if ctx.invoked_subcommand is None:
             await ctx.send_help(ctx.command)
 
+    @cmd_for_bot_stuff.group(name="cogs")
+    @command_log
+    async def embed_available_cogs(self, ctx: commands.Context):
+        """Command handler for the `bot` subcommand `cogs`.
+
+        Creates an Embed containing a list of all available Cogs and their current status (un-/loaded). This embed will
+        then be posted in the configured bot channel.
```

Comment:
Because the embed contains behind the scenes information about which users don't need to know about. This way I make sure, that this embed will never be posted in a public channel.